src/npc.h

```diff
@@ -554,7 +554,7 @@ class npc : public player
  ~npc() override;
 
     /** Construct an NPC with standard stats */
-    static npc standard();
+    static npc make_standard();
```

Comment:
That's better. Yet another option is `make_fake()` (assigns the flag) or even `npc( bool fake = false )` or `npc( bool random = true )` (pros: we don't need to know what "standart" means and why other NPCs aren't standart; cons: passing boolean flags is evil - can be solved with an `enum`). However, the best possible option is getting rid of fake NPCs entirely.